### PR TITLE
Greenthumb (New) - A medium-sized, eco-friendly bar and restaurant

### PR DESCRIPTION
**Pull request by diatomic-ge**
## About the PR

Added a new, eco-friendly restaurant and bar, built to service environmentally-conscious customers in deep space, whether they need a bite to eat, or a drink with friends after a long day mining.

## Why / Balance

I noticed that while there were a few smaller-sized restaurant ships, such as the NC Bocadillo and NC Skipper, and several small/medium ships with a bar, such as the NC Investigator and NT Kestrel, there didn't seem to be many options for both a combined bar and restaurant.

Additionally, I was interested in solar as an option for a deep-space restaurant which didn't need to bring several stacks of plasma or regularly dock with the outpost to refuel, which became the eco-friendly theming.

I tried to use the Skipper and Bocadillo as a rough guide for what equipment to provide.

Apologies for any issues, this is my first ship.

## Media

I couldn't figure out how to get the MapRenderer working, so I stitched together screenshots:
greenthumb

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Wiki Info

*"A self-sufficient, eco-friendly bar and restaurant, featuring its own hydroponics garden."* — The Shipyard
**Ship Size**: Medium
**Recommended Crew**: 1-4
**Cost**: $46000
**Power Gen Type**: Solar, Plasma
**Expeditions**: No
**IFF Console**: No
**Available Roles**: Bartender, Botanist, Chef

**Changelog**

:cl:
- add: Added NC Greenthumb - a medium-sized, eco-friendly bar and restaurant with hydroponics.

### CODE REVIEW

File: Resources/Prototypes/_NF/Shipyard/greenthumb.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  name: NC Greenthumb
+  description: A self-sufficient, eco-friendly bar and restaurant, featuring its own hydroponics garden.
+  price: 46000
+  category: Medium
+  group: Civilian
+  shuttlePath: /Maps/Shuttles/greenthumb.yml
+
+- type: gameMap
+  id: Greenthumb
+  mapName: 'NC Greenthumb'
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/_NF/Shipyard/greenthumb.yml:12]: Remove "NC" from the napName as NC should not appear in the IFF Tag.

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_NF/Shipyard/greenthumb.yml (hunk 1 above)
Comment (diatomic-ge) [reply Resources/Prototypes/_NF/Shipyard/greenthumb.yml]: Sorry about that. Fixed in [b3e6d092](https://github.com/new-frontiers-14/frontier-station-14/pull/796/commits/b3e6d0928ed5a306debd178d5b3d5aeaa1f2e592)

**Comment by Cheackraze** [comment]
i like the idea, the design and layout remind me of just a larger skipper. One thing to note that buttons sharing walls with other interactables or objects is typically a poor practice and can make things difficult to properly interact with. You've got plenty of wall space on the ship, so it would probably be easiest to move the defibs/fire extinguishers and such.  The multi tool has an ability to copy address lists from an existing air alarm, so you can use it to move the air alarm to a differenet place much simpler than having to re-link all the devices.  Again for this one, I'm gunna let magnus give his review as well

**Comment by MagnusCrowe** [comment]
Why blast doors on all the windows?  Expecting to come under siege?

**Comment by iamthememory** [comment]
> Why blast doors on all the windows? Expecting to come under siege?

I liked the idea of being able to close up shop for a bit by closing the shutters and turning off the lights while you're closed, and from other ships, blast doors seemed to be the standard way to do that.

**Comment by iamthememory** [comment]
> i like the idea, the design and layout remind me of just a larger skipper. One thing to note that buttons sharing walls with other interactables or objects is typically a poor practice and can make things difficult to properly interact with. You've got plenty of wall space on the ship, so it would probably be easiest to move the defibs/fire extinguishers and such. The multi tool has an ability to copy address lists from an existing air alarm, so you can use it to move the air alarm to a differenet place much simpler than having to re-link all the devices. Again for this one, I'm gunna let magnus give his review as well

I moved the fire alarm, air alarm, extinguisher cabinet, and defibrillator to their own walls in the customer area.

**Comment by MagnusCrowe** [comment]
Here's a few requests:

1. Replace all the blast doors with shutters.
2. Reduce the late join spawn points from 7 to 1.
3. Consider splitting the power system from one APC into at least two.
4. You have an orphaned piece of HV cable in the dining area you should remove.

**Comment by MagnusCrowe** [comment]
It's been a month with no activity.  I am going to close this PR for now.  Please re-open it when you are ready to come back and work on it. :)

**Pull request closed without merging**